web-app/js/portal/utils/BrowserCheck.js

```diff
@@ -0,0 +1,16 @@
+function browserCheck() {
+	var supported = false; // exclusive
```

Comment:
would it make sense to move this default var assigment to the place where it's actually being used - in the (isChrome || isDirefox) tlogic?